fastcrypto-tbls/src/dkg.rs

```diff
@@ -155,10 +155,16 @@ where
         // to reduce communication.
         let vss_sk = PrivatePoly::<G>::rand(t - 1, rng);
 
-        debug!("Creating party {my_id}, {enc_pk:?} with threshold {t}, {random_oracle:?}");
-        if tracing::enabled!(tracing::Level::DEBUG) {
-            nodes.iter().for_each(|n| debug!("{n:?}"));
-        }
+        let vss_pk = vss_sk.commit::<G>();
+        info!(
+            "DKG: Creating party {}, nodes hash {:?}, t {}, ro {:?}, enc pk {:?}, vss pk c0 {:?}",
+            my_id,
```

Comment:
optional nit - move args inline to format string where possible for better readability (e.g. `my_id`, `t`, etc.)

Reply:
agreed, but since i also had here a value that cannot be inlined, i left everything outside (not sure if better though)